packages/ui/src/forum/modals/PostHistoryModal/PostHistoryModal.tsx

```diff
@@ -24,7 +25,23 @@ export const PostHistoryModal = React.memo(() => {
     modalData: { postId, author },
   } = useModal<PostHistoryModalCall>()
   const { isLoading, edits } = useForumPostEdits(postId)
-  const [activeEdit] = useState(0)
+  const [activeEdit, setActiveEdit] = useState(0)
+
+  const editsInView = useMemo(() => new Array<boolean>(edits?.length ?? 0), [edits?.length])
+
+  const pickActiveEdit = useCallback(
+    (index: number) => (isCurrentInView: boolean) => {
+      editsInView[index] = isCurrentInView
+      const newActiveEdit = editsInView.reduceRight(
+        (currentTopIndex, isEditInView, index) => (isEditInView ? index : currentTopIndex),
+        0
+      )
+      setActiveEdit(newActiveEdit)
+    },
+    [editsInView]
+  )
+
+  const viewport = useRef<HTMLDivElement>(null)
```

Comment:
Seems like the viewport's ref does not change at all. Not sure we need it.

Reply:
I forgot to assign it to the `StepperBody` - I wanted to consider margins so that points light up a bit before edits hit the absolute top of the view